ev-dev-tools/setup.cfg

```diff
@@ -1,6 +1,6 @@
 [metadata]
 name = ev-dev-tools
-version = 0.0.3
+version = 0.0.4
```

Comment:
There is probably room for discussion here, because if the templates change there major version because they would break something, will we need to increase the major version of ev_cli too? Or should ev_cli change major only if the command line interface has breaking changes?

Reply:
That would probably be a good idea. And we should add an easy way to check the ev-cli version (with a `--version` flag), however that's out of the scope of this particular PR.

Reply:
yeah, I guess semantic versioning for all tolls (including ev-cli) would be a good idea